dotCMS/src/integration-test/java/com/dotcms/rendering/velocity/services/HTMLPageAssetRenderedTest.java

```diff
@@ -920,17 +958,14 @@ public void containerArchived_PageShouldResolve() throws Exception {
      * @throws Exception
      */
     @Test
-    public void shouldReturnPageHTMLForPersona() throws Exception{
+    @UseDataProvider("cases")
+    public void shouldReturnPageHTMLForPersona(final Container container, final Template template) throws Exception{
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [A method/constructor should not explicitly throw java.lang.Exception](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5067852)